internal/mux/mux.go

```diff
@@ -22,8 +22,23 @@ func InitalizeRoutes() chi.Router {
 	// Routes for "queue" resource
 	r.Route("/queue", func(r chi.Router) {
 		// POST /articles
-		r.Post("/", handler.CreateQueue)                                        
+		r.Post("/", handler.CreateQueue)
+
+		// Subrouters
+		r.Route("/{id}", func(r chi.Router) {
+			r.Use(handler.QueueCtx)
+			r.Get("/", handler.GetQueue)       // GET /queue/123
+			r.Delete("/", handler.DeleteQueue) // DELETE /queue/123
+		})
+		r.Route("/pause/{id}", func(r chi.Router) {
```

Comment:
Not a expert, should it be `/{id}/pause` or is `/pause/{id}` more correct?

Reply:
Just adding my thoughts here, since it's more of an RPC format API, `/pause?id={id}` makes sense to me as it resembles more of a function call format than the alternatives. But to be fair, I can't really think of any disadvantages in any of the styles. `/{id}/pause` seems a little more intuitive when considering there's a `/{id}` GET endpoint already. What do you think?

Reply:
Shall we try to be more REST-like than RPC line?

Reply:
Either way is fine, I just don't see a reason to have to overthink in terms of what is more or less RESTful. I feel like `/{id}/pause` is the best fit here.

Reply:
changed to /{id}/pause